Podcast/BookmarkViewController.swift

```diff
@@ -15,6 +15,7 @@ class BookmarkViewController: UIViewController, UITableViewDelegate, UITableView
     ///
     var bookmarkTableView: UITableView!
     var cards: [Card] = []
+    var currentlyPlayingIndexPath: IndexPath?
```

Comment:
Unless we reload the data for all cells which would be costly if there are many cards in the table view